src/app/execution/txHelper.js

```diff
@@ -32,20 +32,20 @@ module.exports = {
   },
 
   sortAbiFunction: function (contractabi) {
```

Comment:
Can you please add a comment above here explaining the order? e.g. constructor, fallback, constant functions, non-constant functions?

Reply:
@axic Okay, added now.